examples/classification.jl

```diff
@@ -0,0 +1,158 @@
+# Recreation of https://gpflow.readthedocs.io/en/master/notebooks/basics/classification.html
+
+# %%
+using SparseGPs
+using AbstractGPs
+using GPLikelihoods
+using StatsFuns
+using FastGaussQuadrature
+using Distributions
+using LinearAlgebra
+using DelimitedFiles
+using IterTools
+
+using Plots
+default(; legend=:outertopright, size=(700, 400))
+
+using Random
+Random.seed!(1234)
+
+# %%
+# Read in the classification data
+data_file = pkgdir(SparseGPs) * "/examples/data/classif_1D.csv"
+x, y = eachcol(readdlm(data_file))
+scatter(x, y)
+
+
+# %%
+# First, create the GP kernel from given parameters k
+function make_kernel(k)
+    return softplus(k[1]) * (SqExponentialKernel() ∘ ScaleTransform(softplus(k[2])))
+end
+
+k = [10, 0.1]
+
+kernel = make_kernel(k)
+f = LatentGP(GP(kernel), BernoulliLikelihood(), 0.1)
```

Comment:
Maybe something that deserves better documentation (or an outer constructor with a sensible default value), but LatentGP's `Σy` field (can we find a more sensible name!?) is effectively just the jitter for numeric stability, so should be more something like 1e-6 or so...

Reply:
Looks like it needs at least `1e-4` jitter to work well which is interesting - I'll completely redo the examples in another PR after this one is merged though, so let's discuss it there?